api/envoy/config/filter/http/cache/v2alpha/cache.proto

```diff
@@ -57,7 +57,7 @@ message CacheConfig {
   // contents of a response, as described by
   // https://httpwg.org/specs/rfc7234.html#caching.negotiated.responses.
   //
-  // During insertion, *allowed_vary_headers* acts as a whitelist: if a
+  // During insertion, *allowed_vary_headers* acts as a allowlist: if a
```

Comment:
You might want to also add a rename annotation for `ip_white_list` in client_ssl_auth.proto.

Reply:
Per above I will deal with client_ssl_auth and redis in a follow up.